### PR TITLE
Gh 188/transaction details header view

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SettingsChangeTransactionDetailsHeaderView.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            } else if swapOwner != nil {
+                SwapOwnerTransactionDetailsHeaderView(removedAddress: swapOwner!.prevOwner, addedAddress: swapOwner!.newOwner)
+            } else if setFallbackHandler != nil {
+                SetFallbackHandlerTransactionDetailsHeaderView(address: setFallbackHandler!.handler)
+            } else if enableModule != nil {
+                EnableDisableModuleTransactionDetailsHeaderView(state: .enable, address: enableModule!.module)
+            } else if disableModule != nil {
+                EnableDisableModuleTransactionDetailsHeaderView(state: .disable, address: disableModule!.module)
+            }
+            else if changeThreshold != nil {
```
Comment (DmitryBespalov) [inline Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SettingsChangeTransactionDetailsHeaderView.swift:43]: and the `} else.. `

File: Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SettingsChangeTransactionDetailsHeaderView.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+//  OwnerChangeTransactionHeaderView.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/16/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct SettingsChangeTransactionDetailsHeaderView: View {
```
Comment (DmitryBespalov) [inline Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SettingsChangeTransactionDetailsHeaderView.swift:11]: nice!

File: Multisig/Logic/Ethereum/GnosisSafe.swift [hunk 3]
```diff
@@ -23,6 +23,20 @@ class GnosisSafe {
         ("0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F", "1.1.1")
     ]
 
+    var fallbackHandlers:[(fallbackHandler: Address, label: String)] = [("0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44", "DefaultFallbackHandler")]
```
Comment (sche) [inline Multisig/Logic/Ethereum/GnosisSafe.swift:26]: (App.shared.defaultFallbackHandler, "DefaultFallbackHandler")

File: Multisig/UI/Transactions/TransactionDetails/TransactionStatusTypeView.swift [hunk 4]
```diff
@@ -11,7 +11,7 @@ import SwiftUI
 struct TransactionStatusTypeView: View {
     let transaction: TransactionViewModel
     var body: some View {
-        VStack {
+        VStack (alignment: .leading) {
```
Comment (sche) [inline Multisig/UI/Transactions/TransactionDetails/TransactionStatusTypeView.swift:14]: "VStack (" - extra space

File: Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/EnableDisableModuleTransactionDetailsHeaderView.swift [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+//
+
+import SwiftUI
+
+struct EnableDisableModuleTransactionDetailsHeaderView: View {
+    let state: State
+    var address: Address
+
+    var body: some View {
+        VStack (alignment: .leading, spacing: 11) {
```
Comment (sche) [inline Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/EnableDisableModuleTransactionDetailsHeaderView.swift:16]: extra space

File: Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SetFallbackHandlerTransactionDetailsHeaderView.swift [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct SetFallbackHandlerTransactionDetailsHeaderView: View {
+    let address: Address
+
+    var body: some View {
+        VStack (alignment: .leading, spacing: 11) {
```
Comment (sche) [inline Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SetFallbackHandlerTransactionDetailsHeaderView.swift:15]: and here :)

File: Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SwapOwnerTransactionDetailsHeaderView.swift [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+//
+
+import SwiftUI
+
+struct SwapOwnerTransactionDetailsHeaderView: View {
+    let removedAddress: Address
+    let addedAddress: Address
+
+    var body: some View {
+        VStack (alignment: .leading) {
```
Comment (sche) [inline Multisig/UI/Transactions/TransactionHeaderView/SettingsChangeTransactionHeaderView/SwapOwnerTransactionDetailsHeaderView.swift:16]: and here

File: Multisig/Logic/Ethereum/GnosisSafe.swift (hunk 3 above)
Comment (MouazAlzahabi) [reply Multisig/Logic/Ethereum/GnosisSafe.swift:26]: App.shared.defaultFallbackHandler is intended to be removed

**Pull request merged**